test/e2e/atlas/streams_test.go

```diff
@@ -268,10 +269,23 @@ func TestStreams(t *testing.T) {
 
 		connections := response.GetResults()
 		a := assert.New(t)
-		a.Len(connections, 1)
-		a.Equal(connectionName, connections[0].GetName())
-		a.Equal("Kafka", connections[0].GetType())
-		a.Equal("example.com:8080,fraud.example.com:8000", *connections[0].BootstrapServers)
+		a.Len(connections, 2)
+
+		expected := []struct {
+			Name, Type, BootstrapServers string
+		}{
+			{Name: connectionName, Type: "Kafka", BootstrapServers: "example.com:8080,fraud.example.com:8000"},
+			// sample_stream_solar is a sample connection which is added by default to new stream instances
+			{Name: "sample_stream_solar", Type: "Sample", BootstrapServers: ""},
+		}
+		got := []struct {
+			Name, Type, BootstrapServers string
+		}{
+			{Name: connections[0].GetName(), Type: connections[0].GetType(), BootstrapServers: pointer.GetOrZero(connections[0].BootstrapServers)},
```

Comment:
is this coming from the SDK? if so can you instead do
```suggestion
			{Name: connections[0].GetName(), Type: connections[0].GetType(), BootstrapServers: connections[0].GetBootstrapServers()),
```
same fo rthe other

Reply:
done, thank you